pulser-core/requirements.txt

```
@@ -1,4 +1,5 @@
-jsonschema >= 4.18
+jsonschema >= 4.17.3
```

Technically, we are supporting only 
```suggestion
jsonschema >= 4.18, ==4.17.3
```
It may seem equivalent, but `4.17.4` could be released if there was a bug in 4.17.3.

---

Ah I did not think about that, thanks for noticing !

---

It seems the `,` works as a logical `and`, not `or`. This will work in the event `4.17.4` is released but not `4.17.5`:
```
jsonschema >= 4.17.3, != 4.17.4
```
Perhaps it's a safe assumption to make? The alternative would be to change `LEGACY_JSONSCHEMA`, I'll post a comment with a suggestion.